File: java/src/main/java/com/iteso/nintendo/Tackle.java
package com.iteso.nintendo;

/**
 * Created by satanas on 2/24/18.
 */
public class Tackle implements Attack {
    /**
     * attack damage.
     */
    private int attackDamage;

    /**
     * @return attack name.
     */
    public String attack() {
        return "Tackle";
    }

    /**
     * @return attack damage.
     */
    public int getAttackDamage() {
        return attackDamage;
    }

    /**
     * @param damage attack damage.
     */
    public void setAttackDamage(final int damage) {
        attackDamage = damage;
    }
}

File: python/Tackle.py
class Tackle:
    def __init__(self):
        self.damage = None
        self.name = 'Tackle'

    def attackName(self):
        return self.name

    def attackDamage(self):
        return self.damage

    def setDamage(self, damage):
        self.damage = damage
